### PR TITLE
install keras-nlp and keras-cv pre-release versions

### PR DESCRIPTION
**Pull request by Philmod**
http://b/315833744

### CODE REVIEW

File: Dockerfile.tmpl [hunk 1]
```diff
@@ -549,7 +549,8 @@ RUN pip install flashtext \
         keras-core \
         # b/312946339 latest version not compatible with our version of keras
         keras-cv==0.6.4 \
```
Comment (rosbo) [inline Dockerfile.tmpl:551]: The new `dev` version for keras-cv has also been released. It is `0.8.0.dev0`.

**Review by rosbo — APPROVED** [review]

**Comment by rosbo** [comment]
The tests are failing because it tries to download the weights from `kaggle.com` but internet is disabled in our unit tests for reproducibility / speed / flakiness reasons.

The test sets the `load_weights` parameter to `False` which initialize the model with random weights. This was done to ensure it doesn't try to download the model weights from GCS.

Looks like the new package ignores that parameter... I will ask the keras_nlp folks about it.

https://github.com/Kaggle/docker-python/blob/a42ecffd7b98e0209d94ae74eb9de9bcb891cdb4/tests/test_keras_nlp.py#L24

**Comment by rosbo** [comment]
Before, the "configuration" was checked-in. Now, it needs to load the `config.json` file over the internet and then it can it loads the model with random weights. I guess that is expected now that the config doesn't live in the source code itself. I will think about how to fix this for our test.

**Comment by rosbo** [comment]
@Philmod I pushed to your branch to add a fake API server for kagglehub given internet access is disabled in our smoke tests. I also updated the Dockerfile to install the dev version of `keras-cv`.

**Comment by Philmod** [comment]
Test failure:

```
======================================================================
ERROR: test_fit (test_keras_nlp.TestKerasNLP)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/input/tests/test_keras_nlp.py", line 12, in test_fit
    classifier = keras_nlp.models.BertClassifier.from_preset(
  File "/opt/conda/lib/python3.10/site-packages/keras_nlp/src/models/task.py", line 213, in from_preset
    return super(cls, calling_cls).from_preset(*args, **kwargs)
  File "/opt/conda/lib/python3.10/site-packages/keras_nlp/src/models/task.py", line 190, in from_preset
    tokenizer = load_from_preset(
  File "/opt/conda/lib/python3.10/site-packages/keras_nlp/src/utils/preset_utils.py", line 199, in load_from_preset
    tokenizer.load_assets(asset_dir)
  File "/opt/conda/lib/python3.10/site-packages/keras_nlp/src/tokenizers/word_piece_tokenizer.py", line 338, in load_assets
    self.set_vocabulary(path)
  File "/opt/conda/lib/python3.10/site-packages/keras_nlp/src/models/bert/bert_tokenizer.py", line 92, in set_vocabulary
    super().set_vocabulary(vocabulary)
  File "/opt/conda/lib/python3.10/site-packages/keras_nlp/src/tokenizers/word_piece_tokenizer.py", line 349, in set_vocabulary
    self.vocabulary = [line.rstrip() for line in file]
  File "/opt/conda/lib/python3.10/site-packages/keras_nlp/src/tokenizers/word_piece_tokenizer.py", line 349, in <listcomp>
    self.vocabulary = [line.rstrip() for line in file]
  File "/opt/conda/lib/python3.10/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xc2 in position 3793: ordinal not in range(128)

----------------------------------------------------------------------
```

**Pull request merged**